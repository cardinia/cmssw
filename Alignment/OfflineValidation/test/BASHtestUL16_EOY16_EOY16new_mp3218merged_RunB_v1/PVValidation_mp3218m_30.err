26-Sep-2019 12:34:54 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016B/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018_ver1-v1/90000/2EAB79A5-2A0B-E911-B196-6CC2173BBF20.root
26-Sep-2019 12:34:58 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016B/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018_ver1-v1/90000/2EAB79A5-2A0B-E911-B196-6CC2173BBF20.root
Begin processing the 1st record. Run 272828, Event 71205, LumiSection 1 on stream 0 at 26-Sep-2019 12:35:11.192 CEST
Begin processing the 1001st record. Run 272828, Event 10694291, LumiSection 21 on stream 0 at 26-Sep-2019 12:39:55.779 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 12:42:33 CEST  Run: 272828 Event: 19164238
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 11553
%MSG
Begin processing the 2001st record. Run 272828, Event 21025428, LumiSection 42 on stream 0 at 26-Sep-2019 12:43:18.684 CEST
Begin processing the 3001st record. Run 272936, Event 270637273, LumiSection 189 on stream 0 at 26-Sep-2019 12:46:21.224 CEST
Begin processing the 4001st record. Run 272936, Event 277810825, LumiSection 194 on stream 0 at 26-Sep-2019 12:46:23.220 CEST
Begin processing the 5001st record. Run 272936, Event 286313509, LumiSection 200 on stream 0 at 26-Sep-2019 12:46:25.969 CEST
Begin processing the 6001st record. Run 272936, Event 294768311, LumiSection 206 on stream 0 at 26-Sep-2019 12:46:28.822 CEST
Begin processing the 7001st record. Run 272936, Event 302124505, LumiSection 211 on stream 0 at 26-Sep-2019 12:46:31.360 CEST
26-Sep-2019 12:46:31 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016B/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018_ver1-v1/90000/2EAB79A5-2A0B-E911-B196-6CC2173BBF20.root

=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 OutOfBounds          -w TrackRefitter:Fi                       1        1
    2 fileAction           -s file_close                             1        1
    3 fileAction           -s file_open                              2        2

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 OutOfBounds          272828/19164238                   
    2 fileAction           PostGlobalEndRun                  
    3 fileAction           pre-events       pre-events       

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
Warning                 1                   1
System                  3                   3

dropped waiting message count 0
[8.001MB/8.001MB][100%][==================================================][8.001MB/s]  [8.001MB/8.001MB][100%][==================================================][8.001MB/s]  
