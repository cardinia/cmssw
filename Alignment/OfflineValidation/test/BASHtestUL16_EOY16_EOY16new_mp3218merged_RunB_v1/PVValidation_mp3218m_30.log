000 (3189503.030.000) 09/26 11:21:20 Job submitted from host: <137.138.44.75:9618?addrs=137.138.44.75-9618+[2001-1458-d00-1a--b]-9618&alias=bigbird14.cern.ch&noUDP&sock=schedd_2873_37ae_40>
...
040 (3189503.030.000) 09/26 12:34:27 Started transferring input files
	Transferring to host: <128.142.121.24:9618?addrs=128.142.121.24-9618+[2001-1458-301-b5--100-242]-9618&noUDP&sock=2626_0124_3366>
...
040 (3189503.030.000) 09/26 12:34:27 Finished transferring input files
...
001 (3189503.030.000) 09/26 12:34:27 Job executing on host: <128.142.121.24:9618?addrs=128.142.121.24-9618+[2001-1458-301-b5--100-242]-9618&noUDP&sock=1824_b0d5_3>
...
006 (3189503.030.000) 09/26 12:34:35 Image size of job updated: 24788
	24  -  MemoryUsage of job (MB)
	24100  -  ResidentSetSize of job (KB)
...
006 (3189503.030.000) 09/26 12:39:35 Image size of job updated: 568128
	555  -  MemoryUsage of job (MB)
	567360  -  ResidentSetSize of job (KB)
...
006 (3189503.030.000) 09/26 12:44:36 Image size of job updated: 576212
	562  -  MemoryUsage of job (MB)
	575444  -  ResidentSetSize of job (KB)
...
040 (3189503.030.000) 09/26 12:46:59 Started transferring output files
...
040 (3189503.030.000) 09/26 12:46:59 Finished transferring output files
...
005 (3189503.030.000) 09/26 12:46:59 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:10:29, Sys 0 00:00:05  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:10:29, Sys 0 00:00:05  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	4077  -  Run Bytes Sent By Job
	1018  -  Run Bytes Received By Job
	4077  -  Total Bytes Sent By Job
	1018  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1           
	   Disk (KB)            :     8235        1   2149575 
	   Memory (MB)          :      562     2000      2000 
...
