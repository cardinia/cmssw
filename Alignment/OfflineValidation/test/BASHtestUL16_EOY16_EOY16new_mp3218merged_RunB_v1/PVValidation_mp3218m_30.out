Job started at  Thu Sep 26 12:34:27 CEST 2019
batch dir: /pool/condor/dir_14539 release: /afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/data/commonValidation/alignmentObjects/paconnor/CMSSW_10_6_0/src release base: /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_0
cmsRun PVValidation_mp3218m_272828_cfg.py
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is DATA!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Applying TrackerSurfaceDeformations!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Running DA Algorithm!
Content of working dir is total 8.1M -rw-r--r--. 1 tkello zh 8.1M Sep 26 12:46 PVValidation_mp3218m_272828.root -rw-r--r--. 1 tkello zh 20K Sep 26 12:34 PVValidation_mp3218m_272828_cfg.py -rw-r--r--. 1 tkello zh 2.8K Sep 26 12:46 _condor_stderr -rw-r--r--. 1 tkello zh 522 Sep 26 12:34 _condor_stdout -rwxr-xr-x. 1 tkello zh 1018 Sep 26 12:34 condor_exec.exe -rw-------. 1 tkello zh 4.3K Sep 26 12:34 tkello.cc drwx------. 3 tkello zh 4.0K Sep 26 12:34 tmp drwx------. 3 tkello zh 4.0K Sep 26 12:34 var
Job ended at  Thu Sep 26 12:46:59 CEST 2019
